test/integration/test_federalist_redirects.js

```diff
@@ -24,6 +24,7 @@ const expectedRedirects = [
   { from: 'www.digital.gov', to: 'digital.gov', redirectCode: 301 },
   { from: 'summit.digitalgov.gov', to: 'digital.gov' },
   { from: 'openopps.digitalgov.gov', to: 'openopps.usajobs.gov', redirectCode: 301 },
+  { from: 'blogging-guide.18f.gov', to: 'handbook.18f.gov/blogging', redirectCode: 301, noPath: true },
```

Comment:
should this have a / as well give your edit to line 75 in federalist-redirects?

Reply:
the ending slash is in the testing function .. https://github.com/18F/pages-redirects/blob/b4a8c357ca0b0ba2b5ce8571b5e95fced2175bc1/test/integration/test_federalist_redirects.js#L44